pages/tenant-administration/classroom-types.vue

```diff
@@ -0,0 +1,118 @@
+<script setup lang="ts">
+import { IconBaseline, IconEdit, IconKey, IconTrash, IconUser, IconZoom } from '@tabler/icons-vue'
+
+const account = useAccount()
+await account.get()
+
+if (!account.data?.roles.includes('TenantAdministrator'))
+  navigateTo('/')
+
+const classroomTypes = useClassroomTypes()
+await classroomTypes.get()
+
+const { currentItem, createDialog, deleteDialog, handleCreate, handleDelete, handleDialogOpen, handleUpdate, isSubmitting, search, updateDialog } = useCrud(classroomTypes.data)
+</script>
+
+<template>
+  <div class="flex w-full flex-wrap items-center justify-between gap-4 border-b border-b-gray-200 px-12 py-9">
+    <div>
+      <h1 class="text-2xl font-bold leading-9 text-gray-900">
+        Typy zajęć
+      </h1>
+    </div>
+
+    <div class="flex gap-4">
+      <base-input v-model="search" placeholder="Szukaj" class="w-96" :icon="IconZoom" />
+      <base-button class="h-12" variant="primary" @click="handleDialogOpen('create')">
+        Dodaj typ zajęć
+      </base-button>
+    </div>
+  </div>
+
+  <base-table :data="classroomTypes.data" :columns="classroomTypes.columns" :search="search">
+    <div class="mb-4 flex items-center">
+      <input
+        id="default-checkbox" type="checkbox" value=""
+        class="size-4 rounded border-gray-300 bg-gray-100 text-blue-600 focus:ring-2 focus:ring-blue-500 dark:border-gray-600 dark:bg-gray-700 dark:ring-offset-gray-800 dark:focus:ring-blue-600"
+      >
+      <label for="default-checkbox" class="ms-2 text-sm font-medium text-gray-900 dark:text-gray-300">Default
+        checkbox</label>
+    </div>
+
+    <template #name="{ cell }">
+      <span class="text-base font-medium text-gray-900">{{ cell.name }}</span>
+    </template>
+
+    <template #isPrimitiveType="{ cell }">
+      <base-checkbox v-model="cell.isPrimitiveType" label="" :disabled="true" />
```

Comment:
Jeśli label może być pusty, to niech jego typ to będzie ```string?```

Reply:
Poza tym, domyślnie binduje się true jeżeli booleanowy prop jest obecny
```suggestion
      <base-checkbox v-model="cell.isPrimitiveType" />
```

Reply:
Co do diasbled to domyślnie jest false, sprawdziłem. label usuwam.

Reply:
Domyślnie wszystkie zmienne boolowskie mają false. Bo kiedyś w C wartość bół powstała z wartości int, gdzie 0 to false, a dowolna inna wartośc to 1. A domyślnie nieprzypisany int ma 0.

I taki standard jest.